src/components/Footer.jsx

```diff
@@ -0,0 +1,19 @@
+import {Text} from '@chakra-ui/react'
+function Footer() {
+    
+    return(
+        <div style={{
+            backgroundColor:'#787878',
+            width:'100%',
+            height:'120px',
+            display: 'flex',
+            flexDirection:'column',
+            
+        }}>
+            <Text fontSize='md' textAlign='center' paddingTop='20px' color='white' >¿TIENES ALGÚN PROBLEMA, DUDA O COMENTARIO?</Text>
+            <Text fontSize='md' textAlign='center' paddingTop='5px' color='white'>NO DUDES EN CONTACTARNOS</Text>
+            <Text fontSize='md' textAlign='center' paddingTop='10px' color='white'>CORREO@GMAIL.COM</Text>
```

Comment:
Puede ser buena idea poner un mailto
```suggestion
            <Text fontSize='md' textAlign='center' paddingTop='10px' color='white'><a href="mailto:correo@gmail.com">CORREO@GMAIL.COM</a></Text>
```